src/sentry/receivers/users.py

```diff
@@ -6,12 +6,18 @@
 from sentry.models import User
 
 
-def create_first_user(created_models, verbosity, db, app=None, **kwargs):
+def create_first_user(verbosity, using=None, db=None, app=None, app_config=None, **kwargs):
```

Comment:
Do we still need the `verbosity` argument for django=1.6?

Reply:
What do you mean?

Also no you should only have to run things once. Was this from a clean install or south?

Reply:
answered my own question on the verbosity part